store/mockstore/unistore/cophandler/closure_exec.go

```diff
@@ -486,12 +546,14 @@ func (e *countColumnProcessor) Process(key, value []byte) error {
 		}
 		if !isNull {
 			e.rowCount++
+			gotRow = true
 		}
 	}
 	return nil
 }
 
 func (e *countColumnProcessor) Finish() error {
+	e.aggCtx.execDetail.updateOnlyRows(1)
```

Comment:
Why not update `timeProcessed` here?

Reply:
Because other `Processor` only update `timeProcessed` in function `Process`, which not include `Finish`